runners/core-java/src/main/java/org/apache/beam/runners/core/construction/SerializablePipelineOptions.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.util.construction;
+package org.apache.beam.runners.core.construction;
```

Comment:
What about just re-introduce this to the flink runner? Yes the package doesn't match but at least it is isolated and not where anyone else might use it.

Reply:
This was my initial line of thinking, but due to discussion on the ML it ended as generic class. It it actually true that the same problem might apply to other runners as well.